services/device-registry/src/main/java/org/eclipse/hono/deviceregistry/ApplicationConfig.java

```diff
@@ -206,6 +207,17 @@ public DeviceManagementHttpEndpoint registrationHttpEndpoint() {
         return new DeviceManagementHttpEndpoint(vertx());
     }
 
+    /**
+     * Creates an authentication handler used by device registry management HTTP API.
+     *
+     * @return The handler.
+     */
+    @Bean
+    @Scope("prototype")
+    public AuthHandler authHandler() {
```

Comment:
same here

Reply:
It was used for manual testing and I thought it'd be good to leave it as a placeholder for the future. I removed it as suggested.